src/admin/getting_started_checks/design_removeDefaultBannersFrontPage.php

```diff
@@ -49,6 +49,7 @@ public function isComplete()
     {
         $name = 'front_page.tpl';
         $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-        return (strpos($custom, '<a href="#"><img src="{external file=\'images/banners/banner2.jpg\'}" alt="" /></a>') === false);
+        if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) return false;
```

Comment:
Single line if's should be avoided, and they should always have the brackets.  If's without the brackets lead to bugs.  Like this instead:
```php
if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) {
    return false;
}
return true;
```

Also, something I forgot to mention that I try to always do before making a commit:

Start up a container command line in web-8.0 container and run this:

`composer cs-fix`

That may make changes to fix any PSR things (basically to have a consistent "code style"), so if it makes any changes, include then in the PR.

On a side note, if we ever get `cs-check` to pass without errors, we can make it part of the tests, so in this case I believe the cs-check code sniffer would have pointed out the if statement without brackets thing.

Reply:
Also I just noticed, you are just checking for `1and1_300x100.jpg` but the code you are replacing is checking for another file, was that intentional?

Reply:
Yes. If you go back and check the original frontpage template, it references 3 different banner images, none of which the checklist script test for. The script was broken in v18. It works now.

Reply:
> Start up a container command line in web-8.0 container and run this:
> 
> composer cs-fix
> 
hmmm, not sure this was such a good idea. I now have 711 changed files showing in my Fix-Issue-#81 branch and when I spot check a few, they all say "No content changes found."

cs-fix says: A TOTAL OF 715 ERRORS WERE FIXED IN 712 FILES

No way I am committing this. What should I do now?

Reply:
I don't know what Github Desktop for Windows is smoking, but I want nothing to do with it. I selected the 710 files I did not change and told GDW to 'stash' them. Well, it decides to stash everything, even the 1 I told it not to. Then when I go to look at the files it stashed, there are only 7 of them. Yep! 7, not 711.

There is the one I changed and 6 others. They all have some funky stuff after the </html>. See pic below.

In any case, I am committing the change that fixes the if () {} issue. Please accept the changes. I am begging you 🤣

![making-me-crazy](https://user-images.githubusercontent.com/6634041/156691454-c63d5042-c099-44ce-b9af-e2bd308ee258.JPG)

Reply:
Speaking of cs-fix. Not sure why it is "fixing" 1 issue and leaving many more in the same file??

![Capture](https://user-images.githubusercontent.com/6634041/156693274-e2d50cd3-1b40-4a8a-9616-d5b5cb3264bf.JPG)

Reply:
> The script was broken in v18. It works now. This is why I put those developer comments in the templates.

Ah OK, so that falls under "intentional" that it is different than before.  👍 

> No way I am committing this. What should I do now?

Go ahead and leave it off without being scanned, I'll use #119 to fix it so you can use it going forward.